Dev/soeren Unified Data Sets

soerenehrig opened the pull request:
Hab die Labels von den Data-sets mal unified dass sie konsistente Nummern in den yolo Format haaben. Außerdem hab ich eine conver.py funktion für roadsigns hizugefügt damit diese auch im Yolo Format sind

soerenehrig committed:
Changed deepdrive/convert.py to have unified Labels with kitti and roadsigns. Added roadsigns/convert.py to convert xml to yolo format and also have it have unified labels

soerenehrig committed:
Updated Data Readme to show unified Labels

squarra commented:
Das `convert.py` skript funktioniert bei mir nicht wie es soll. Die Fotos haben alle unterschiedliche größen, die auch aus der XML ausgelesen werden müssen. 

Um zu testen ob die labels korrekt sind, kannst du einen training run starten und dir die dateien `train_batch*.png` anschauen, um zu sehen ob die boxen an der richtigen stelle sind.

```sh
python3 train.py ../../data/roadsigns/config.yaml
```

squarra commented:
Wir könnten auch das notebook abändern, sodass es nicht mehr die "annotations" nutzt um die bilder zu zeichnen, sondern das "labels" directory. Dann würde man sowas direkt sehen.

squarra commented:
Oder allgemein einfach sowas: #15

soerenehrig commented:
Ich würde erstmal es ändern und mit nem training run überprüfen aber für zukünftig kann #15 schon sinvoll

soerenehrig committed:
- Updated Convert Function (Roadsigns) to split Images and Labels correctly into Folders.
- Updated Config.yaml files to have the same labeling for (deepdrive,kitty,roadsigns)

squarra commented:
Ich glaube das funktioniert immer noch nicht so ganz xD

Im images/val directory sind andere Dateien als im labels/val. Irgendwas geht bei der zufälligen Auswahl der Nummern kaputt oder so.

soerenehrig committed:
Smal Fixes in the roadsigns\README.md

squarra commented:
Ok ja bei mir klappts auch
